### ISSUE TITLE
Allow "Disable notification for this conversation" to remove the notification

### ISSUE THREAD

**Issue by lapineige** [tootsuite/mastodon#11373]
### Pitch
We can mute notification from one conversation (each reply to one toot where we are mentioned).
But that only works for new messages.

This action could (optionally*) remove already existing replies to one toot from the notifications panel.

* as a question popup, in the same way as you can ask to confirm if you want to delete a toot: you can either only disable newer notifications, or remove existing ones.

### Motivation

Sometimes you can come back to mastodon after a while, and people mentioned you a lot in an unwanted conversation (shitposting, trolling, spam, off-topic discussion and so on).
You could want to remove those unwanted notifications from the list.

This would help to clean up the notifications panel as well as reducing some trolling/spamming effect.

**Comment by lapineige** [comment]
No one thinks it's a good idea ? :smile:

**Comment by CecileVillemin** [comment]
Yeah, that's a good idea

**Comment by CecileVillemin** [comment]
Yeah, that's a good idea. It would avoid having to browse through all the notifications of a conversation from the moment it was hidden
